packages/clerk-js/src/ui/components/SignUp/SignUpContinue.tsx

```diff
@@ -150,6 +152,9 @@ function _SignUpContinue() {
     card.setLoading();
     card.setError(undefined);
 
+    const redirectUrl = buildSSOCallbackURL(ctx, displayConfig.signUpUrl);
+    const redirectUrlComplete = ctx.afterSignUpUrl || '/';
```

Comment:
🙃 I believe we already properly resolve this inside the `RedirectUrls` class
```suggestion
    const redirectUrlComplete = ctx.afterSignUpUrl
```